### PR TITLE
US121769 Accessibility Improvements

### PR DESCRIPTION
**Pull request by Vitalii-Misechko**
[US121769](https://rally1.rallydev.com/#/23525809118d/iterationstatus?detail=%2Fuserstory%2F449072958256&sharedViewId=328666453148&view=dc9e7043-1cf6-486d-a4aa-3a212976e789&fdp=true?fdp=true): [Engagement] Accessibility Improvements

FYI @MykolaGalian @rohitvinnakota @hyehayes @NicholasHallman @johngwilkinson 

AC:
  1. Make sure the table works as expected with screen readers
    * Compared LMS's Users table and dashboard's User Details table. Applied minor changes.
    * Dashboard's Active Courses table looks fine to me.
  2. Enable eslint a11y rules https://github.com/open-wc/open-wc/tree/master/packages/eslint-plugin-lit-a11y

* Functional Testing
  * [x] Browsers (Chrome, FF, Edge)
  * [x] Accessibility (NVDA, JAWS)
  * [x] Test cases
    * User navigates to Email buttons of User Details table. Screen reader: _Table actions. Email button_ (JAWS says also: _To activate hit enter/space_)
    * User navigates to User Details table. Screen reader: _User Details table. This table lists the Users that satisfy the filtering criteria. Actions that may be applied to multiple items are available in the table actions._ (+ description of the table columns and the first column)
* Security, devops, perf: N/A
* UX and docs
  * [x] Demo

### CODE REVIEW

File: components/expander-with-control.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable lit-a11y/click-events-have-key-events */
```
Comment (Vitalii-Misechko) [inline components/expander-with-control.js:1]: The component works fine with keyboard, thus, added ignore comment for the file.

File: locales/en.js [hunk 2]
```diff
@@ -18,6 +18,7 @@ export default {
 	"dashboard:learMore": "Learn More",
 	"dashboard:exportToCsv": "Export to CSV",
 	"dashboard:saveDefaultView": "Make this my default view",
+	"dashboard:emailButtonAriaLabel": "Table actions. Email",
```
Comment (anhill-D2L) [inline locales/en.js:21]: In the classlist, their email button has the aria label `Email selected users in a new window` and reads as `Clickable email selected users in a new window button`. Something similar might be good for consistency

Comment (Vitalii-Misechko) [reply locales/en.js:21]: Cool! I'll update

Comment (Vitalii-Misechko) [reply locales/en.js]: Done

**Review by anhill-D2L — APPROVED** [review]

**Review by bemailloux — APPROVED** [review]

**Pull request merged**